### PR TITLE
Remove Prisma and Update Commands in Debug Your Astronomer Software Installation

### LINKED ISSUE #227: 0.25 and 0.23 Debugging your cluster are incorrect for postgresql usage
@rbankston commented on [Mon Jun 21 2021](https://github.com/astronomer/docs-archived/issues/393)

**Affected Document:**
  https://www.astronomer.io/docs/enterprise/v0.25/troubleshoot/debug-install and https://www.astronomer.io/docs/enterprise/v0.23/troubleshoot/debug-install

**Severity:**

- [ ] Critical: This document has incorrect or missing information. Add the `error/critical` label to this issue.
- [x] FYI: The document is misleading or could benefit from additional context or guidance. Add the `error/FYI` label to this issue.
- [ ] Typo: Add the `error/typo` label to this issue.
  
**Suggested Change (include links to relevant existing docs, support tickets, messages, or external content):**

Page looks like it comes from 0.16 when houston ran as root. Houston now only runs as houston. You can exec into one of the stan or nats pods to run the same commands as the root user.

**Additional Notes:**

### PR DESCRIPTION
**Pull request by kwcanuck**
Resolves #227

### CODE REVIEW

**Review by rbankston — APPROVED** [review]
looks good to me

File: software/debug-install.md [hunk 1]
```diff
@@ -44,50 +44,20 @@ manageable-snail-kubed-5b5d65dd9d-l7nds                    1/1     Running
...
 ```
 
-If these are the only three pods that are not coming up as healthy, it is usually indicative of an issue with connecting to the database. Try checking:
+If these pods do not come up in a healthy state, it is usually an issue with the database connection. Try checking:
 
 #### Networking
-Make sure that the Kubernetes cluster Astronomer is running on can connect to the database. One way to check this is to jump into an Astronomer pod and try connecting directly to the database:
+Make sure that the Kubernetes cluster Astronomer is running on can connect to the database. Run the following comannd to start a `postgresql` pod in your cluster and then connect to it:
 
 ```
```
Comment (jwitz) [inline software/debug-install.md:56]:
```suggestion
```sh
```

File: software/debug-install.md [hunk 2]
```diff
@@ -44,50 +44,20 @@ manageable-snail-kubed-5b5d65dd9d-l7nds                    1/1     Running
 manageable-snail-nginx-799d79ccf9-kfnzn                    1/1     Running            0          1h
 manageable-snail-nginx-default-backend-5cc4755696-vh5zq    1/1     Running            0          1h
 manageable-snail-astro-ui-7b9b9df4f9-pb99f                 1/1     Running            0          1h
-manageable-snail-prisma-6b5d944bdc-szn8f                   0/1     CrashLoopBackOff   20         1h
 manageable-snail-prometheus-0                              1/1     Running            0          1h
 manageable-snail-registry-0                                1/1     Running            0          1h
 ```
 
-If these are the only three pods that are not coming up as healthy, it is usually indicative of an issue with connecting to the database. Try checking:
+If these pods do not come up in a healthy state, it is usually an issue with the database connection. Try checking:
```
Comment (jwitz) [inline software/debug-install.md:51]:
```suggestion
If these pods do not come up in a healthy state, it is usually an issue with the database connection. See the following topics to confirm your connection.
```
Feel free to mess with the wording, but I thought having a colon leading into a header could give the impression that there's missing content before the header.

**Review by jwitz — APPROVED** [review]
Two minor comments, otherwise LGTM!

**Pull request merged**